backend/src/main/java/com/example/demo/todo/TodoService.java

```diff
@@ -27,11 +27,15 @@ public Optional<Todo> findById(String id) {
 
     public Optional<Todo> updateTodo(String id, String email, Todo updatedTodo) {
                 Optional<Todo> todoFromDB = todoRepository.findByIdAndUserEmail(id, email);
-                // abfrafe zum verglichen kommt hier
-                        return
+                int currentScore = 0;
+                if(updatedTodo.getStatus() != todoFromDB.get().getStatus()){
+                } currentScore++;
```

Comment:
du hast da einen kleinen Fehler: die `}` muss nach `currentScore++;` schließen, so passiert nichts in dem if und jedes mal wird currentScore höher gesetzt

Reply:
eingentlich nicht. erst wenn guckt wird, ob der status anders ist, dann wird es berechnet. Alle todos fangen mit open status an. Erst, wenn es auf done gesetzt wird, dann wird hier was berechnet